cmake/FindOpenSCAP.cmake

```diff
@@ -0,0 +1,41 @@
+SET(OpenSCAP_POSSIBLE_ROOT_DIRS
+    "${OPENSCAP_ROOT_DIR}"
+    "$ENV{OPENSCAP_ROOT_DIR}"
+    "$ENV{ProgramFiles}"
+    "/usr/local/share"
+    "/usr/share/"
+    "/usr/local"
+    "/usr"
+    "/usr/bin"
+    "/usr/sbin"
+    "/opt"
+    "/opt/local"
+  )
+
+FOREACH(NAME ${OpenSCAP_POSSIBLE_ROOT_DIRS})
+    FIND_PATH(OpenSCAP_XSL_DIR NAMES xccdf_1.1_to_1.2.xsl
+    PATHS "${NAME}"
+    PATH_SUFFIXES "share/openscap/xsl"
```

Comment:
Nitpick for consistency, this suffix has not trailing slash. Suffix for `bin` has a trailing slash.